mailpoet/lib/Cron/CronTrigger.php

```diff
@@ -51,6 +52,7 @@ public function init(string $environment = '') {
       return false;
     } catch (\Exception $e) {
       // cron exceptions should not prevent the rest of the site from loading
+      Helpers::mySqlGoneAwayExceptionHandler($e);
```

Comment:
Just want to make sure I understand. Here, we intentionally ignore the returned message and call the helper just to log the error via error_log. Correct?

Reply:
Yup, that's correct

Reply:
Oh, if we have a Database error, we can't write to the Database, so it probably won't work.